frictionless_ckan_mapper/ckan_to_frictionless.py

```diff
@@ -1,5 +1,7 @@
 # coding=utf-8
 import json
+import re
+import unidecode
```

Comment:
This should be added as a requirement

Reply:
I added the unidecode to setup.py. re is in the standard library, I thought unidecode was also, sorry.